fault_injection/configs/pen.global_fi.otbn.char.dmem.access.cw310.yaml

```diff
@@ -0,0 +1,43 @@
+  plot_x_axis_legend: "[cycles]"
+  plot_y_axis: "glitch_width"
+  plot_y_axis_legend: "[cycles]"
+test:
+  which_test: "otbn_char_dmem_access"
+  expected_result: '{"res":0,"data":[1752170618,3418933566,771316262,4010052340,2483697855,667193667,4142343277,258919171,1568367398,2083390144,822779226,1460736182,2472577306,1592644756,3462497725,2727813326,4031028770,2013723758,893979386,1311732921,1428641407,4120548177,4056821378,4020034490,2386760574,3343172890,2293757165,2262723144,748249686,1401292733,807848246,3730046934,2619074860,2032943536,1085649462,2671920459,1622914787,1912311726,3391365577,4256333020,2233440140,1440072100,2102472527,4219498733,3628925464,2016031521,496835627,2100551899,1954377022,1322324621,4133891729,2763881761,2199950666,3696470864,223672220,1879127539,3154836533,3664663838,2994899182,464953814,2783894004,7829331,3278890329,1538924890,1667118926,411767882,1164559549,1652414630,3064362610,2023222856,1688489707,308279992,64438546,2854554286,3758103254,3165512448,3555303949,1496873998,1089353835,4285534125,1223513568,974584991,2344229968,766618146,1942701401,1139369403,75412658,52415182,2255817440,3526010056,2536578112,3569007285,2081641394,1516189644,265179347,813865141,747614792,168407506,108663204,4292691240,794894046,2237659246,377116394,820426910,1638793281,3097826007,3558814448,960610601,1930477156,1671537109,250494443,2586076563,946256704,3267121137,3189486529,3964200644,888992584,3495652898,1529411913,2598903086,3513281746,3231295608,2803605408,3100054487,645745023,2355743283,1480423879,2172027945,1266942366,3913525525,1092007188,623760003,1328680366,1682389994,2525476757,91894463,999873799,233141538,3150195604,548149341,3209425774,3888891766,2765385373,3567370867,3377184759,3299905540,3255715595,3329689998,2822181885,3284805046,4229950334,825123374,1038376279,2470646264,4218773437,3746395077,1448724516,3262565001,1661122272,1877452977,857694292,772569637,802843685,2125455361,3406777056,1764727446,920060882,4025369693,1349100798,2657217694,3394126751,687787461,576621678,1479550633,2357376939,1082544701,2007372129,1976310578,3156904125,592833811,3705765941,359181864,2221615505,881896932,4111468059,3200275311,2017758330,2983828617,2458693710,1318909501,761916836,3033935028,3438888195,3544373064,49464920,28038751,3663653666,762100395,1461355421,2489968999,365663631,1685236111,1515268868,3440084261,3377173686,2531354775,3364444857,355319372,3977655639,2562537247,2632720970,2322265413,4165295828,1717227322,191431321,3846398043,1421641274,1737581554,899970312,3106177760,3829243285,3798641989,1615864806,3367735900,3225082500,4154605851,3694669111,3189306406,2529953284,1801951682,1348275831,559991528,3520078732,3214041459,2057586075,2195673018,3287101809,2946758789,3811293796,2432574270,4201292549,1274171431,1009532021,1331756015,4233976805,1172338183,3572168806,2894879080,4008636142,117835012,185207048,252579084,319951120,387323156,454695192,522067228],"insn_cnt":271,"err_otbn":0,"err_ibx":0,"alerts":[0,0,0]}'
```

Comment:
Question: is the data section of the expected result here the readable part of DMEM? Are there any dependencies on RND/URND, scrambling keys or similar that could prevent successful repeating / porting of the test? Sidenote: on FPGA, the entropy source is not really random.

Reply:
In this test, this is the expected data. This data is defined in the DMEM section in the [test](https://github.com/nasahlpa/opentitan/blob/pentest/sw/device/tests/penetrationtests/firmware/fi/otbn/otbn_char_dmem_access.s) and it is read out by Ibex.

But when thinking about your comment, I've actually found out that one test, the CHAR.BN.WSRR test indeed fetches random data. I've added a comment in the config file and created a new issue #382.

Reply:
Thanks for the explanation, this sounds good to me!